encoders/ffmpeg_test.go

```diff
@@ -140,7 +140,7 @@ var _ = Describe("FFmpeg Encoder", func() {
 					},
 				},
 				Status:           types.JobCreated,
-				Details:          "0%",
```

Comment:
I think it's fine. However, it would be great to add the video with no audio (http://flv.io/video.mp4) on `fixtures/` folder and run a test that breaks the encoding, just to assert that the `Details` field is being used for that. What do you think?